store/store.go

```diff
@@ -92,6 +94,12 @@ func (s *Store) Start() error {
 	return s.TxManager.ActivateAccount(acc)
 }
 
+// Stop shuts down all of the working parts of the store.
+func (s *Store) Stop() error {
+	close(s.RunChannel)
```

Comment:
This will likely panic unless you can be certain that all the `Perform` goroutines are finished.